src/ErrorHandler/JsonRenderer.php

```diff
@@ -7,6 +7,13 @@
 final class JsonRenderer extends ThrowableRenderer
 {
     public function render(\Throwable $t): string
+    {
+        return json_encode([
```

Comment:
could we use `yiisoft/json`?

Reply:
Yes but I see no benefit of doing it in this particular case.